pkg/controller/gitopsconfig/controller.go

```diff
@@ -239,9 +276,40 @@ func (r *Reconciler) createCronJob(instance *gitopsv1alpha1.GitOpsConfig) error
 
 	if err != nil {
 		log.Error(err, "unable to create/update the cronjob", "cronjob", cronjob)
-		return err
+		return "", err
 	}
-	return nil
+	time.Sleep(5 * time.Second)
+	var result batchv1beta1.CronJob
+	err = r.client.Get(context.TODO(),
+		types.NamespacedName{Name: cronjob.Name, Namespace: cronjob.Namespace},
+		&result)
+
+	if err != nil {
+		log.Info("Error in GetCronJob:", err)
+		return "", err
+	}
+	runningJobArr := result.Status.Active
+	length := len(runningJobArr)
+	if length < 1 {
+		return "FAILED", nil
+	}
+	latestJobRunning := runningJobArr[len(runningJobArr)-1]
```

Comment:
The [kubernetes docs for CronJobStatus](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.11/#cronjobstatus-v1beta1-batch) describe `active` field as: "A list of pointers to currently running jobs." I think this is a good idea to check the contents of `active` here. However, I don't understand how do we know that it is enough to check just the last of them?